.github/workflows/release_channels.yml

```
@@ -63,29 +66,42 @@ jobs:
       - name: Get release version string
         run: |
           echo "long=$(./bin/buildinfo.py long)" >> $GITHUB_OUTPUT
+          echo "short=$(./bin/buildinfo.py short)" >> $GITHUB_OUTPUT
           echo "deb=$(./bin/buildinfo.py deb)" >> $GITHUB_OUTPUT
         id: version
         env:
           BUILD_LOCATION: local
 
       - name: Bump version.properties
-        run: >-
-          bin/bump_version.py
+        run: |
+          # Bump version.properties
+          chmod +x ./bin/bump_version.py
+          ./bin/bump_version.py
 
       - name: Ensure debian deps are installed
-        shell: bash
         run: |
           sudo apt-get update -y --fix-missing
           sudo apt-get install -y devscripts
 
       - name: Update debian changelog
-        run: >-
-          debian/ci_changelog.sh
+        run: |
+          # Update debian changelog
+          chmod +x ./debian/ci_changelog.sh
+          ./debian/ci_changelog.sh
+
+      - name: Bump org.meshtastic.meshtasticd.metainfo.xml
+        run: |
+          # Bump org.meshtastic.meshtasticd.metainfo.xml
+          pip install -r bin/bump_metainfo/requirements.txt -q
+          chmod +x ./bin/bump_metainfo/bump_metainfo.py
+          ./bin/bump_metainfo/bump_metainfo.py --file bin/org.meshtastic.meshtasticd.metainfo.xml "${{ steps.version.outputs.short }}"
 
-      - name: Create version.properties pull request
+      - name: Create Bumps pull request
         uses: peter-evans/create-pull-request@v7
         with:
-          title: Bump version.properties
+          title: "Bump: Bumps on Published release"
```

```suggestion
          title: "Bump release version"
```
I'd try to keep this short and sweet :) and use the word "version" in the title so it's easier to understand.